### PR TITLE
Allow using --ci flag on annotate subcommands

### LINKED ISSUE #230: Allow using --ci flag on individual annotators
It would be really nice to be able to do something similar to this, so that one does not need to enforce all annotation types during CI runs:
```
bin/cake annotate models -v -d --ci
bin/cake annotate controllers -v -d --ci
```

### PR DESCRIPTION
**Pull request by ekisu**
Closes #230 

I couldn't find a good way to test the `view` subcommand when there are no changes, I'm open to suggestions on that

### CODE REVIEW

File: tests/TestCase/Shell/AnnotationsShellTest.php [hunk 1]
```diff
@@ -190,7 +236,7 @@ public function testClasses() {
 	public function testCallbacks() {
 		$result = $this->Shell->runCommand(['callbacks', '-d', '-v']);
 
-		$this->assertNull($result);
+		$this->assertEquals(0, $result);
```
Comment (dereuromark) [inline tests/TestCase/Shell/AnnotationsShellTest.php:239]: AssertSame

**Comment by dereuromark** [comment]
Whats the problem with the view one?

**Comment by ekisu** [comment]
> Whats the problem with the view one?

`view` does not support using the `--plugin` parameter, which is what was used to test the CI mode when used with the `all` subcommand. I tried using `--filter` to just pick a file with no problems, but it doesn't seem to be supported as well, so I'm kinda out of ideas here.

**Comment by dereuromark** [comment]
Maybe it is fair enough for now as is.

**Comment by dereuromark** [comment]
Anything missing or should we merge this?

**Comment by ekisu** [comment]
It's fine for me

**Pull request merged**